docs/rackspace-elastic-engineering-and-optimizer/index.rst

```diff
@@ -0,0 +1,226 @@
+collaborating side-by-side with highly skilled Rackspace Technology cloud
+experts to build, migrate, optimize, and maintain the customer’s cloud
+environments.
+
+The following table provides a sample of the types of services provided with
+Rackspace Elastic Engineering. We coordinate tasks, scheduling, and priorities
+between Rackspace Technology and the customer.
+
+.. list-table::
+   :widths: 40 110
+   :header-rows: 1
+
+   * - Service
+     - Description
+   * - Infrastructure Builds
+     - * Deploy infrastructure or applications into the environment
+       * Modify infrastructure to reduce cost, increase performance, and scale optimization
+   * - DevOps and Automation
+     - * Manage, maintain, and enhance automation systems for operational effectiveness, stability and security
+       * Change existing automation systems, or develop new automation systems
+   * - Performance Management
+     - * Identify performance targets and continuously adjust infrastructure to deliver the desired performance at the optimal cost
+       * Tune applications, provide detailed analysis of application-level performance problems
+   * - Database Management
+     - * Scale databases, provide performance management and maintenance services
+       * Create database clusters, read slaves, and so on
+   * - Application Operations
+     - * Manage the configuration and patching of common, off-the-shelf applications
+       * Provide custom upgrades and patches for off-the-shelf applications, develop configuration frameworks and methodologies for custom applications
+   * - Security Management
+     - * Use architectural best practices to maintain environment security
+       * Respond to security incidents identified by the Customer Security Team
+       * Deploy customer-provided third-party security tools
+       * Create reports or contribute to reports that a customer needs as part of an infrastructure audit for certification purposes
+   * - Disaster Recovery Planning
+     - * Run disaster recovery solutions, extend existing solutions, and run customer-directed tests of disaster recovery systems
+   * - Planned Maintenance
+     - * Perform preventative maintenance so that infrastructure, database, and application operations are stable and secure
+       * Perform optional maintenance and customer-directed changes, which can include applying security patches on a regular basis
+   * - Cost Optimization
+     - * Implement architectural changes to support cost management decisions
+   * - Advice and Support
+     - * Answer questions related to cloud usage, best practices, and make recommendations
+
+.. _monitoring_resolution:
+
+Advanced Monitoring and Resolution
+-----------------------------------
+
+Built for customers' critical workloads and always vigilant 24x7x365, our team
+of operations engineers monitors and responds to alerts in customer
+environments. Advanced Monitoring and Resolution (AMR) is an optionated
```

Comment:
Spelling:

`s/optionated/opinionated/`

Reply:
that's our new slogan - it means that we use our expertise to make decisions about how a customer should deploy into the cloud

Reply:
I don't think `optionated` is a word though 😄 

It _seems_ like it should be `opinionated`